packages/components/config/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../config/tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../lib"
+  },
+  "include": ["../src/**/*"],
+  "exclude": ["../src/**/__tests__/**/*", "../src/**/__mocks__/**/*"]
+}
```

Comment:
Same comment: shouldn't this `tsconfig.json` file live in the root `packages/components` directory?

Reply:
See above comment.